### PR TITLE
Added support for bower

### PR DESCRIPTION
**Pull request by gabrielbull**

### CODE REVIEW

**Comment by yamadapc** [comment]
Thanks!

Why use bower, though? You'll need something like `webpack` or `browserify` to handle building CommonJS regardless... This module has no dependencies, but uses `require` internally, so installing it with bower and adding it to an HTML file won't work.

If you handle that I might merge this, otherwise I'll close it tomorrow.

**Comment by gabrielbull** [comment]
Yeah, I figured it out after. I'll check this today.

**Comment by gabrielbull** [comment]
Okay this works in the browser now.

File: lib/index.js [hunk 1]
```diff
@@ -1,120 +1,133 @@
-'use strict';
```
Comment (yamadapc) [inline lib/index.js]: I'm pretty sure you don't have to wrap it in a closure and that browserify will do it for us

**Comment by gabrielbull** [comment]
I removed the closure

File: lib/index.js [hunk 2]
```diff
@@ -118,3 +117,14 @@ function handleSmallerThan100(n, language, unit, baseCardinals, options) {
   }
   return baseCardinals[dec];
 }
+
```
Comment (yamadapc) [inline lib/index.js]: You also don't need this, just add the standalone option.

Comment (gabrielbull) [inline lib/index.js]: Can you explain, how is it supposed to work in the browser without this part?

Comment (yamadapc) [inline lib/index.js]: `--standalone writtenNumber` will do it for you.

**Comment by gabrielbull** [comment]
Alright, I added the standalone option.

**Comment by yamadapc** [comment]
Beautiful! Thank you very much!

**Pull request merged**